proto-demo: {
  sequence1: [rest],
  sequence_a: [pick-right, rest-right],
  sequence_b: [pick-left, rest-left],
  sequence_x: [rest],
  sequence_y: [rest],
  sequence_LB: [rest],
  sequence_RB: [rest],
  sequence_back:  [rest],
  sequence_start: [home],
  sequence_power: [rest],
  sequence_left_stick:  [left-gripper-open, left-gripper-half],
  sequence_right_stick: [right-gripper-open, right-gripper-half],
  
  right-gripper-open:   { right_gripper_joint: {joint_num: 5,   joint_angle: 1.1779}  },
  right-gripper-half:   { right_gripper_joint: {joint_num: 5,   joint_angle: 0}       },
  right-gripper-closed: { right_gripper_joint: {joint_num: 5,   joint_angle: -1.1779} },
  left-gripper-open:    { left_gripper_joint:  {joint_num: 11,  joint_angle: 1.1779}  },
  left-gripper-half:    { left_gripper_joint:  {joint_num: 11,  joint_angle: 0}       },
  left-gripper-closed:  { left_gripper_joint:  {joint_num: 11,  joint_angle: -1.1779} },
  
  home: {
      right_armroot_to_armlink1:  {joint_num: 0,  joint_angle: 0.0234},
      right_armlink1_to_armlink2: {joint_num: 1,  joint_angle: 1.5572},
      right_armlink2_to_armlink3: {joint_num: 2,  joint_angle: -1.1096},
      right_armlink3_to_armlink4: {joint_num: 3,  joint_angle: -1.5439},
      right_armlink4_to_armlink5: {joint_num: 4,  joint_angle: -1.5707},
      left_armroot_to_armlink1:   {joint_num: 6,  joint_angle: 0.0234},
      left_armlink1_to_armlink2:  {joint_num: 7,  joint_angle: -1.5572},
      left_armlink2_to_armlink3:  {joint_num: 8,  joint_angle: -1.1096},
      left_armlink3_to_armlink4:  {joint_num: 9,  joint_angle: 1.5439},
      left_armlink4_to_armlink5:  {joint_num: 10, joint_angle: -1.5707}
  },
  
  home-right: {
      right_armroot_to_armlink1:  {joint_num: 0,  joint_angle: 0.0234},
      right_armlink1_to_armlink2: {joint_num: 1,  joint_angle: 1.5572},
      right_armlink2_to_armlink3: {joint_num: 2,  joint_angle: -1.1096},
      right_armlink3_to_armlink4: {joint_num: 3,  joint_angle: -1.5439},
      right_armlink4_to_armlink5: {joint_num: 4,  joint_angle: -1.5707}
  },
  
  home-left: {
      left_armroot_to_armlink1:   {joint_num: 6,  joint_angle: 0.0234},
      left_armlink1_to_armlink2:  {joint_num: 7,  joint_angle: -1.5572},
      left_armlink2_to_armlink3:  {joint_num: 8,  joint_angle: -1.1096},
      left_armlink3_to_armlink4:  {joint_num: 9,  joint_angle: 1.5439},
      left_armlink4_to_armlink5:  {joint_num: 10, joint_angle: -1.5707}
  },
  
  pick: {
      right_armroot_to_armlink1:  {joint_num: 0,  joint_angle: 0},
      right_armlink1_to_armlink2: {joint_num: 1,  joint_angle: -1.5707},
      right_armlink2_to_armlink3: {joint_num: 2,  joint_angle: 0.7854},
      right_armlink3_to_armlink4: {joint_num: 3,  joint_angle: 1.5707},
      right_armlink4_to_armlink5: {joint_num: 4,  joint_angle: 1.3464},
      left_armroot_to_armlink1:   {joint_num: 6,  joint_angle: 0},
      left_armlink1_to_armlink2:  {joint_num: 7,  joint_angle: 1.5707},
      left_armlink2_to_armlink3:  {joint_num: 8,  joint_angle: 0.7854},
      left_armlink3_to_armlink4:  {joint_num: 9,  joint_angle: -1.5707},
      left_armlink4_to_armlink5:  {joint_num: 10, joint_angle: 1.3464}
  },
  
  pick-right: {
      right_armroot_to_armlink1:  {joint_num: 0,  joint_angle: 0},
      right_armlink1_to_armlink2: {joint_num: 1,  joint_angle: -1.5707},
      right_armlink2_to_armlink3: {joint_num: 2,  joint_angle: 0.7854},
      right_armlink3_to_armlink4: {joint_num: 3,  joint_angle: 1.5707},
      right_armlink4_to_armlink5: {joint_num: 4,  joint_angle: 1.3464}
  },
  
  pick-left: {
      left_armlink1_to_armlink2:  {joint_num: 7,  joint_angle: 1.5707},
      left_armlink2_to_armlink3:  {joint_num: 8,  joint_angle: 0.7854},
      left_armlink3_to_armlink4:  {joint_num: 9,  joint_angle: -1.5707},
      left_armlink4_to_armlink5:  {joint_num: 10, joint_angle: 1.3464}
  },
  
  rest: {
      right_armroot_to_armlink1:  {joint_num: 0,  joint_angle: 0.7853},
      right_armlink1_to_armlink2: {joint_num: 1,  joint_angle: -1.5705},
      right_armlink2_to_armlink3: {joint_num: 2,  joint_angle: -1.5705},
      right_armlink3_to_armlink4: {joint_num: 3,  joint_angle: -1.5705},
      right_armlink4_to_armlink5: {joint_num: 4,  joint_angle: -0.5235},
      left_armroot_to_armlink1:   {joint_num: 6,  joint_angle: -0.7853},
      left_armlink1_to_armlink2:  {joint_num: 7,  joint_angle: 1.5705},
      left_armlink2_to_armlink3:  {joint_num: 8,  joint_angle: -1.5705},
      left_armlink3_to_armlink4:  {joint_num: 9,  joint_angle: 1.5705},
      left_armlink4_to_armlink5:  {joint_num: 10, joint_angle: -0.5235}
  },
  
  rest-right: {
      right_armroot_to_armlink1:  {joint_num: 0,  joint_angle: 0.7853},
      right_armlink1_to_armlink2: {joint_num: 1,  joint_angle: -1.5705},
      right_armlink2_to_armlink3: {joint_num: 2,  joint_angle: -1.5705},
      right_armlink3_to_armlink4: {joint_num: 3,  joint_angle: -1.5705},
      right_armlink4_to_armlink5: {joint_num: 4,  joint_angle: -0.5235}
  },
  
  rest-left: {
      left_armroot_to_armlink1:   {joint_num: 6,  joint_angle: -0.7853},
      left_armlink1_to_armlink2:  {joint_num: 7,  joint_angle: 1.5705},
      left_armlink2_to_armlink3:  {joint_num: 8,  joint_angle: -1.5705},
      left_armlink3_to_armlink4:  {joint_num: 9,  joint_angle: 1.5705},
      left_armlink4_to_armlink5:  {joint_num: 10, joint_angle: -0.5235}
  },

  up: {
      right_armroot_to_armlink1:  {joint_num: 0,  joint_angle: 0},
      right_armlink1_to_armlink2: {joint_num: 1,  joint_angle: 0},
      right_armlink2_to_armlink3: {joint_num: 2,  joint_angle: 0},
      right_armlink3_to_armlink4: {joint_num: 3,  joint_angle: 0},
      right_armlink4_to_armlink5: {joint_num: 4,  joint_angle: 0},
      left_armroot_to_armlink1:   {joint_num: 6,  joint_angle: 0},
      left_armlink1_to_armlink2:  {joint_num: 7,  joint_angle: 0},
      left_armlink2_to_armlink3:  {joint_num: 8,  joint_angle: 0},
      left_armlink3_to_armlink4:  {joint_num: 9,  joint_angle: 0},
      left_armlink4_to_armlink5:  {joint_num: 10, joint_angle: 0}
  },
  
  up-right: {
      right_armroot_to_armlink1:  {joint_num: 0,  joint_angle: 0},
      right_armlink1_to_armlink2: {joint_num: 1,  joint_angle: 0},
      right_armlink2_to_armlink3: {joint_num: 2,  joint_angle: 0},
      right_armlink3_to_armlink4: {joint_num: 3,  joint_angle: 0},
      right_armlink4_to_armlink5: {joint_num: 4,  joint_angle: 0}
  },
  
  up-left: {
      left_armroot_to_armlink1:   {joint_num: 6,  joint_angle: 0},
      left_armlink1_to_armlink2:  {joint_num: 7,  joint_angle: 0},
      left_armlink2_to_armlink3:  {joint_num: 8,  joint_angle: 0},
      left_armlink3_to_armlink4:  {joint_num: 9,  joint_angle: 0},
      left_armlink4_to_armlink5:  {joint_num: 10, joint_angle: 0}
  }
}
